cmd/controller/controller.go

```diff
@@ -232,11 +232,16 @@ func (c *command) start(ctx context.Context) error {
 		if c.SingleNode {
 			return errors.New("control plane load balancing cannot be used in a single-node cluster")
 		}
+		if cplb.Type != v1beta1.CPLBTypeKeepalived {
```

Comment:
IMO we should do this in config validation already.

Reply:
Agreed, this is done in a separate commit because rewriting the git history was quite complex...